images/win/scripts/Installers/Install-Msys2.ps1

```diff
@@ -9,35 +9,22 @@
 
 $dash = "-" * 40
 
-$origPath = $env:PATH
-$gitPath  = "$env:ProgramFiles\Git"
-
-$msys2_release = "https://api.github.com/repos/msys2/msys2-installer/releases/latest"
-
-$msys2Uri = ((Invoke-RestMethod $msys2_release).assets | Where-Object {
+# Downloading msys2
+$msys2Release = "https://api.github.com/repos/msys2/msys2-installer/releases/latest"
+$msys2Uri = ((Invoke-RestMethod $msys2Release).assets | Where-Object {
   $_.name -match "x86_64" -and $_.name.EndsWith("tar.xz") }).browser_download_url
```

Comment:
MSYS2 provides a self-extracting executable, apart from `tar.xz` tarballs. The content is the same. Hence, I'd suggest to use the self-extracting solution, which avoids the need for 7z.  See https://github.com/msys2/setup-msys2/blob/master/main.js#L12 and https://github.com/msys2/setup-msys2/blob/master/main.js#L222-L223. This is the commit where it was changed: https://github.com/msys2/setup-msys2/commit/03af082796cc8420caff534b796ff2c4830ee83c.

Reply:
Thank you. Done.